src/blockchain/vulcan0x.ts

```diff
@@ -38,11 +78,16 @@ export function vulcan0x(
     },
     body: {
       query:
-      '{\n' +
-      `    ${resource}(\n${options}\n){\n` +
+      `query ${id}${queryParams} {\n` +
+      `    ${resource}(\n${options}\n) {\n` +
       `      nodes { ${fields.join(' ')} }\n` +
       '    }\n' +
-      '}'
+      '}',
+      variables: {
+        ...fromPairs(variables.map(({ name, value }) => [name, value])),
+        ...process.env.REACT_APP_GRAPHQL_DEVMODE ? { devMode: process.env.REACT_APP_GRAPHQL_DEVMODE } : {},
```

Comment:
devMode is a secret? maybe it should be named `DEVMODE_SECRET`?

Reply:
Should we really announce that this is a secret?

Reply:
> We shouldn't announce the secret value but the mechanism itself is public.

Reply:
We will keep this mechanism private.